### DISCUSSION TITLE
Suppress warnings when running rome check

### DISCUSSION THREAD

**Discussion by gudleik** [rome/tools#3548] [Suggestions]
We have some symlinked files in our repo, and running rome check generates alot of noise like

```
apps/desktop/public/favicon.svg internalError/fs
⚠ Symbolic links are not supported
ℹ Rome does not currently support visiting the content of symbolic links since it could lead to an infinite traversal loop
```

Would be nice to be able to suppress these warnings.

**Comment by ematipico** [comment]
That's a good suggestion, we definitely need to provide a way to silent some diagnostics

**Comment by Nickersoft** [comment]
Was about to open a feature request as well till I caught this one – I agree this feature is desperately needed. I've swapped over a number of errors to warnings, as my codebase is rather large, and trying to parse through the sea of warnings to figure out which errors were thrown that caused Rome to fail is very very painful 😭
